tests/upgrades/test_errata.py

```diff
@@ -298,6 +308,12 @@ def test_post_scenario_errata_count_installtion(self):
         for errata in FAKE_9_YUM_ERRATUM:
             host.errata_apply(data={'errata_ids': [errata]})
             applicable_errata_count -= 1
+        wait_for(
+            lambda: self._errata_count(ak=activation_key) == 0,
+            timeout=200,
+            delay=10,
+            logger=self.logger
+        )
```

Comment:
Overall logic looks good to me .
It would look great,  if could define this as : https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_yum_plugins.py#L143-L156

and use as https://github.com/SatelliteQE/robottelo/blob/master/tests/upgrades/test_yum_plugins.py#L209

Reply:
As per my knowledge, there is no need to write a new function here to add more processing calls. By the name itself, it signifies it is mean for wait till errata count becomes 0. Even if you think, I can add a comment above this to make sure that get understandable to the code reader

Reply:
The idea here is to mention the reason to wait on this task here .
It's fine to add a comment here.